### PR TITLE
Gyro button invert and toggle features

### PR DESCRIPTION
**Pull request by lukacsi**
Implements: partially #70, fully #387 
### Require button now can be set to the following values in settings:

- ON: legacy behavior, if you press the gyro button you get gyro controls
- INVERT: you get gyro controls, if you press the gyro button you have stick controls
- TOGGLE: toggle between gyro and stick controls
- OFF: legacy behavior, you get gyro controls regardless of the button

Added descriptions for the values, the same way as GyroYawMode
Tested on 1.21-fabric with DualSense controller on linux

### CODE REVIEW

File: src/main/java/dev/isxander/controlify/ingame/InGameInputHandler.java [hunk 1]
```diff
@@ -313,7 +316,11 @@ protected void handleRegularLook(InputComponent input, Vector2f impulse, boolean
     protected void handleGyroLook(GyroComponent gyro, Vector2f impulse, boolean aiming) {
         GyroComponent.Config config = gyro.confObj();
 
-        if (config.requiresButton && (!ControlifyBindings.GYRO_BUTTON.on(controller).digitalNow() && !aiming)) {
+        if (config.requiresButton.equals(GyroButtonMode.ON) && (!ControlifyBindings.GYRO_BUTTON.on(controller).digitalNow() && !aiming)) {
```
Comment (isXander) [inline src/main/java/dev/isxander/controlify/ingame/InGameInputHandler.java:319]: maybe separate all these usages of `ControlifyBindings.GYRO_BUTTON.on(controller)` into a local called gyroButton to make the code a little cleaner!

**Review by isXander — CHANGES_REQUESTED** [review]

File: src/main/java/dev/isxander/controlify/ingame/InGameInputHandler.java (hunk 1 above)
Comment (lukacsi) [reply src/main/java/dev/isxander/controlify/ingame/InGameInputHandler.java]: made the changes

**Review by isXander — APPROVED** [review]

**Comment by isXander** [comment]
Hey, if you have Discord, lmk your username and I shall add you to the code contributor role!

**Comment by lukacsi** [comment]
Thanks, my user name is ogergec.

Xander ***@***.***> ezt írta (időpont: 2024. okt. 29., K,
14:00):

**Pull request merged**